Index: konsole/src/main.cpp
===================================================================
--- konsole.orig/src/main.cpp
+++ konsole/src/main.cpp
@@ -282,6 +282,9 @@ void fillCommandLineOptions(QCommandLine
     parser.addOption(QCommandLineOption(QStringList() << "p",
                                         i18nc("@info:shell", "Change the value of a profile property."),
                                         QStringLiteral("property=value")));
+    parser.addOption(QCommandLineOption(QStringList() << "T",
+                                        i18nc("@info:shell", "Debian policy compatibility, not used."),
+                                        QStringLiteral("value")));
     parser.addOption(QCommandLineOption(QStringList() << "e",
                                         i18nc("@info:shell", "Command to execute. This option will catch all following"
                                               " arguments, so use it as the last option."),
