From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 12:17:59 +0200
Subject: debian-T-addition

===================================================================
---
 src/main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/main.cpp b/src/main.cpp
index 5e1420e..b1792d5 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -232,6 +232,7 @@ void fillCommandLineOptions(KCmdLineOptions& options)
                       " (for use with -p)"));
     options.add("p <property=value>",
                 ki18nc("@info:shell", "Change the value of a profile property."));
+    options.add("T <value>", ki18n("Debian policy compatibility, not used"));
     options.add("!e <cmd>",
                 ki18nc("@info:shell", "Command to execute. This option will catch all following"
                       " arguments, so use it as the last option."));
