From: "Martin T. H. Sandsmark" <martin.sandsmark@kde.org>
Date: Sun, 15 Jan 2017 14:14:10 +0100
Subject: Don't include newlines in selection if trimming is on

BUG: 199381
REVIEW: 129837
---
 src/Screen.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/Screen.cpp b/src/Screen.cpp
index 5438e83b..0b3124a5 100644
--- a/src/Screen.cpp
+++ b/src/Screen.cpp
@@ -1185,7 +1185,8 @@ void Screen::writeToStream(TerminalCharacterDecoder* decoder,
         // this makes it possible to 'select' a trailing new line character after
         // the text on a line.
         if (y == bottom &&
-                copied < count) {
+                copied < count &&
+                !trimTrailingSpaces) {
             Character newLineChar('\n');
             decoder->decodeLine(&newLineChar, 1, 0);
         }
@@ -1257,7 +1258,7 @@ int Screen::copyLineToStream(int line ,
             // ignore trailing white space at the end of the line
             for (int i = length-1; i >= 0; i--)
             {
-                if (data[i].character == ' ')
+                if (QChar(data[i].character).isSpace())
                     length--;
                 else
                     break;
