From: "Martin T. H. Sandsmark" <martin.sandsmark@kde.org>
Date: Sun, 22 Jan 2017 18:20:54 +0100
Subject: Screen: Fix potential crash when ExtendedCharTable tries to clean

---
 src/Screen.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Screen.h b/src/Screen.h
index aaa282eb..443d0ecf 100644
--- a/src/Screen.h
+++ b/src/Screen.h
@@ -580,7 +580,7 @@ public:
         QSet<ushort> result;
         for (int i = 0; i < _lines; ++i) {
             const ImageLine& il = _screenLines[i];
-            for (int j = 0; j < _columns; ++j) {
+            for (int j = 0; j < il.length(); ++j) {
                 if (il[j].rendition & RE_EXTENDED_CHAR) {
                     result << il[j].character;
                 }
