From: Anthony Fieroni <bvbfan@abv.bg>
Date: Mon, 6 Feb 2017 21:20:00 +0200
Subject: [TerminalDisplay] Abort painting invalid rect

REVIEW: 129914

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>
---
 src/TerminalDisplay.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/TerminalDisplay.cpp b/src/TerminalDisplay.cpp
index 1d70d7c2..f8263499 100644
--- a/src/TerminalDisplay.cpp
+++ b/src/TerminalDisplay.cpp
@@ -1749,7 +1749,8 @@ void TerminalDisplay::updateCursor()
 
 void TerminalDisplay::resizeEvent(QResizeEvent*)
 {
-    updateImageSize();
+    if (contentsRect().isValid())
+        updateImageSize();
 }
 
 void TerminalDisplay::propagateSize()
